chat_rag_explorer/__init__.py

```
@@ -53,12 +53,20 @@ def _log_startup_config(app):
-    masked_key = mask_api_key(api_key)
+    provider_name = app.config.get("LLM_PROVIDER", "openrouter")
+    if provider_name == "ollama":
+        base_url = app.config.get("OLLAMA_BASE_URL", "NOT SET")
+        api_key = app.config.get("OLLAMA_API_KEY", "")
+        key_env_var = "OLLAMA_API_KEY"
+    else:
+        base_url = app.config.get("OPENROUTER_BASE_URL", "NOT SET")
+        api_key = app.config.get("OPENROUTER_API_KEY", "")
+        key_env_var = "OPENROUTER_API_KEY"
 
     logger.info(f"Configuration:")
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)